lib/lost-align.js

```diff
@@ -130,17 +135,12 @@ module.exports = function lostAlignDecl(css, settings) {
       }
     } else {
       if (lostAlign == 'reset') {
-        decl.cloneBefore({
-          prop: 'display',
-          value: 'initial'
+        cloneAllBefore({
+          'display': 'initial',
+          'display': 'flex',
```

Comment:
Is there a reason display is set to `initial` then `flex` here?

Could this just be 
```js
cloneAllBefore({
  'display': 'flex',
  'justify-content': 'inherit',
  'align-items': 'inherit'
});